### PR TITLE
Issue #3311: [DX][Docs] Update the API documentation to include the new 'info' type for messages.

### PR DESCRIPTION
**Pull request by klonos**
https://github.com/backdrop/backdrop-issues/issues/3311

### CODE REVIEW

File: core/includes/bootstrap.inc [hunk 1]
```diff
@@ -2321,12 +2321,22 @@ function watchdog($type, $message, $variables = array(), $severity = WATCHDOG_NO
  *   (optional) The translated message to be displayed to the user. For
  *   consistency with other messages, it should begin with a capital letter and
  *   end with a period.
+ * @param string $message
```
Comment (opi) [inline core/includes/bootstrap.inc]: redundant

File: core/includes/bootstrap.inc [hunk 2]
```diff
@@ -2321,12 +2321,22 @@ function watchdog($type, $message, $variables = array(), $severity = WATCHDOG_NO
...
+ *   (optional) The translated message to be displayed to the user. For
+ *   consistency with other messages, it should begin with a capital letter and
+ *   end with a period.
  * @param string $type
- *   (optional) The message's type. Defaults to 'status'. These values are
+ *   (optional) The type of the message. Defaults to 'status'. These values are
```
Comment (opi) [inline core/includes/bootstrap.inc:2325]: Not sure about informal tone in documentation. I personnaly don't mind, but we should be consistant across the whole documentation.

**Review by opi — CHANGES_REQUESTED** [review]

File: core/includes/bootstrap.inc (hunk 2 above)
Comment (klonos) [reply core/includes/bootstrap.inc:2325]: ...oops 😅...fixed!

Comment (klonos) [reply core/includes/bootstrap.inc:2325]: Yep, not sure either. That's why I am asking for feedback.

**Comment by klonos** [comment]
Closing in favor of https://github.com/backdrop/backdrop/pull/2575

**Pull request closed without merging**